haas/cli.py

```diff
@@ -247,17 +247,24 @@ def serve_networks():
     """Start the HaaS networking server"""
     from haas import model, deferred
     from time import sleep
+    import ConfigParser
     server.init()
     server.register_drivers()
     server.validate_state()
     model.init_db()
     migrations.check_db_schema()
+    # Check if config contains usable sleep_time
+    try:
+        sleep_time = float(cfg.get('devel', 'sleep_time'))
+    except (ConfigParser.NoSectionError, ConfigParser.NoOptionError,
+            ValueError):
+        sleep_time = 2
```

Comment:
I'd rather not just silently use the default value if the user makes a typo in their config file, e.g.

```
[devel]
sleep_time  = e3 # user fat-fingered, hit two keys at once
```

And, re: handling the absent case, it would be clearer to just do:

```python
if cfg.has_option('devel', 'sleep_time'):
    sleep_time = cfg.get('devel', 'sleep_time')
else:
    ...
```